app/src/main/java/com/yogeshpaliyal/comrade/utils/PackageVerifier.kt

```
@@ -0,0 +1,74 @@
+object PackageVerifier {
+
+
+    /**
+     * Verifies if a package is valid by matching its certificate with the previously stored
+     * certificate.
+     */
+    fun getCertificate(context: Context, packageName: String): String? {
+        val hash: String
+        try {
+            return getCertificateHash(context, packageName)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)